src/content/en/2019/markup.md

```diff
@@ -0,0 +1,301 @@
+
+<table>
+  <tr>
+    <td>2005</td>
+    <td>2019</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td></td>
+  </tr>
+</table>
+
+
+## Custom elements?
+
+Most of the elements we recorded are custom (as in simply 'not standard'), but discussing which elements are and are not custom can get a little challenging. Written down in some spec or proposal somewhere are, actually, quite a few elements.  For purposes here, we considered 244 elements as standard (though, some of them are deprecated or unsupported):
+
+* 145 Elements from HTML
+* 68 Elements from SVG
+* 31 Elements from MathML
+
+In practice, we encountered only 214 of these:
+
+* 137 from HTML
+* 54 from SVG
+* 23 from MathML
+
+In the desktop dataset we collected data for the top 4,834 non-standard elements that we encountered. Of these:
+
+* 155 (3.21%) are identifiable as very probable markup or escaping errors (they contain characters in the parsed tag name which imply that the markup is broken)
+* 341 (7.05%) use XML-style colon namespacing (though, as HTML, they don't use actual XML namespaces)
+* 3207 (66.44%) are valid custom element names
+* 1211 (25.05%) are in the global namespace (non-standard, having neither dash, nor colon)
+    * 216 of these we have flagged as *possible *typos as they are longer than 2 characters and have a Levenshtein distance of 1 from some standard element name like `<cript>`,`<spsn>` or `<artice>`. Some of these (like `<jdiv>`), however, are certainly intentional.
+
+Additionally, 15% of desktop pages and 16% of mobile pages contain deprecated elements (NOTE:  A lot of this is very likely due to the use of products rather than individual authors continuing to manually create this markup.), here's the most common 10 and the number of pages they appear on in each set...
+
+<table>
+  <tr>
+    <td>element</td>
+    <td>mobile</td>
+    <td>desktop</td>
+  </tr>
+  <tr>
+    <td><center></td>
+    <td>7.96%</td>
+    <td>8.30%</td>
+  </tr>
+  <tr>
+    <td><font></td>
+    <td>7.80%</td>
+    <td>8.01%</td>
+  </tr>
+  <tr>
+    <td><marquee></td>
+    <td>1.20%</td>
+    <td>1.07%</td>
+  </tr>
+  <tr>
+    <td><nobr></td>
+    <td>0.55%</td>
+    <td>0.71%</td>
+  </tr>
+  <tr>
+    <td><big></td>
+    <td>0.47%</td>
+    <td>0.53%</td>
+  </tr>
+  <tr>
+    <td><frame></td>
+    <td>0.35%</td>
+    <td>0.39%</td>
+  </tr>
+  <tr>
+    <td><frameset></td>
+    <td>0.39%</td>
+    <td>0.35%</td>
+  </tr>
+  <tr>
+    <td><strike></td>
+    <td>0.27%</td>
+    <td>0.32%</td>
+  </tr>
+  <tr>
+    <td><noframes></td>
+    <td>0.27%</td>
+    <td>0.25%</td>
+  </tr>
+</table>
+
+
+Most of these can seem like very small numbers, but perspective matters.
+
+## Perspective on Value and Usage
+
+In order to discuss numbers about the use of elements (standard, deprecated or custom), we first need to establish some perspective.
```

Comment:
I still think extrapolating to "all websites" can help with getting a perspective. Comparing different percentages with each other doesn't help so much. Saying something like

> There are around [2 billion web sites on the web](https://www.websitehostingrating.com/internet-statistics-facts/). If something appears on 0.1% of all websites, that represents *2 million web sites*.

And then maybe go on to say that this is why HTML elements are *very* rarely removed from the web platform -- breaking a million or so web sites isn't something browser vendors do lightly.

Reply:
Ok, I do like the idea of some extrapolation like this that stresses the idea that these are big numbers.

Reply:
Added to line 206 of the current pull

> So, are all of those elements used by less than 1% of pages "useless?".  Definitely not.  This is why establishing perspective matters.  There are around [2 billion web sites on the web](https://www.websitehostingrating.com/internet-statistics-facts/). If something appears on 0.1% of all websites in our dataset, we can extrapolate that this represents perhaps *2 million web sites* in the whole web. Even 0.01% extrapolates to a _two hundred of thousand of sites_.  This is also why removing support for elements, even very old ones which we think aren't great ideas, is a very rare occurrence:  Breaking hundreds of thousands or millions of sites just isn't a thing that browser vendors can do lightly.